src/displayapp/screens/settings/SettingWatchFace.cpp

```diff
@@ -65,7 +66,7 @@ void SettingWatchFace::UpdateSelected(lv_obj_t* object, lv_event_t event) {
     for (unsigned int i = 0; i < options.size(); i++) {
       if (object == cbOption[i]) {
         lv_checkbox_set_checked(cbOption[i], true);
-        settingsController.SetClockFace(i);
+        settingsController.SetClockFace(static_cast<WatchFace>(i));
```

Comment:
Maybe SetClockFace() should recieve a WatchFace and do the cast internally?

Reply:
The cast is from an uint8_t to a WatchFace, right? SetClockFace already takes a WatchFace.

Reply:
Oh, I missed that. Sorry.